bundle/Core/Pagination/Pagerfanta/SearchTagsAdapter.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Netgen\TagsBundle\Core\Pagination\Pagerfanta;
+
+use eZ\Publish\API\Repository\Exceptions\UnauthorizedException;
+use Netgen\TagsBundle\API\Repository\TagsService;
+use Pagerfanta\Adapter\AdapterInterface;
+
+class SearchTagsAdapter implements AdapterInterface, SearchTagsAdapterInterface
+{
+    /**
+     * @var \Netgen\TagsBundle\API\Repository\TagsService
+     */
+    protected $tagsService;
+
+    /**
+     * @var string
+     */
+    protected $searchTerm;
+
+    /**
+     * @var string
+     */
+    protected $language;
+
+    /**
+     * @var int
+     */
+    protected $nbResults;
+
+    /**
+     * @param \Netgen\TagsBundle\API\Repository\TagsService $tagsService
+     */
+    public function __construct(TagsService $tagsService)
+    {
+        $this->tagsService = $tagsService;
+    }
+
+    /**
+     * @param string $searchTerm
+     */
+    public function setSearchTerm(string $searchTerm)
```

Comment:
Can't use scalar typehints. 3.4 branch still supports PHP 5.6

Reply:
Done in https://github.com/netgen/TagsBundle/commit/f4a8ea03a3e319d60e3f799889a2fd0b12598249